src/xb-string.c

```diff
@@ -178,6 +178,32 @@ xb_string_search (const gchar *text, const gchar *search)
 	return FALSE;
 }
 
+/**
+ * xb_string_search: (skip)
+ * @text: The source string
+ * @search: The text to search for
```

Comment:
This should probably say that `search` is actually a (`NULL` terminated?) array of somethings

Reply:
I've fixed up that and given an example.